src/main/webapp/app/entities/submission.model.ts

```diff
@@ -62,6 +62,18 @@ export function getLatestSubmissionResult(submission: Submission | undefined): R
     }
 }
 
+/**
+ * Used to access a submissions result for a specific correctionRound
+ *
+ * @param submission
+ * @returns the results or undefined if submission or the result for the requested correctionRound is undefined
+ */
+export function getSubmissionResultByCorrectionRound(submission: Submission | undefined, correctionRound: number): Result | undefined {
```

Comment:
I am a little bit confused why you are allowing here that the function can be called with an undefined as submission?

Reply:
It was added as  a fallback, to make the function as flexible as possible. Will be removed soon if its not necessary.